llvm/lib/CodeGen/GlobalISel/CombinerHelper.cpp

```diff
@@ -6810,9 +6791,6 @@ bool CombinerHelper::matchSelect(MachineInstr &MI, BuildFnTy &MatchInfo) {
   if (tryFoldBoolSelectToLogic(Select, MatchInfo))
     return true;
 
-  if (tryFoldSelectToIntMinMax(Select, MatchInfo))
```

Comment:
It was invoked for many selects in the function and probably failed a lot for no win.